README.md

```diff
@@ -175,10 +183,8 @@ $ rm -rf tanka
 
 **Issue:** ./install.sh fails complaining about Ingress. You're using a `k3d` that runs a too new version of k8s. That means that Ingress got promoted and the `extensions.v1beta1.ingress` in [production/sample/default/main.jsonnet] is not valid anymore.
 
-**Solution:** You are likely using a version of `k3d` that is too new. Most likely you are using a newer version of `k3d` which runs a k8s version that no longer supports Ingress under `extensions.v1beta1`. Verify it running kubectl version. As per [this blogpost](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) k8s v1.22 and higher no longer serve that API. To solve the issue, use an older version of `k3d` which will run an older version of k8s. For example, version 5.0.0 should work (runs v1.21.5).
-
-
-## Contributing guidelines
+**Solution:** You're using a newer version of `k3s`, which means that the `k8s` version it provides no longer supports Ingress under `extensions.v1beta1`. Verify it running `kubectl version`. As per [this blogpost](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) k8s v1.22 and higher no longer serve that API. To solve the issue, use an older version of `k3d` which will run an older version of k8s. For example, version 5.0.0 should work (runs v1.21.5).
```

Comment:
not sure why the "Commit suggestion" throws a 500 since yesterday, so I will manually introduced your changes. I'll merge it as soon as the build is green.